openstack/dds/v3/instances/requests.go

```diff
@@ -22,7 +22,6 @@ type CreateOpts struct {
 	Mode                string          `json:"mode" required:"true"`
 	Configuration       []Configuration `json:"configurations,omitempty"`
 	Flavor              []Flavor        `json:"flavor" required:"true"`
-	BackupStrategy      BackupStrategy  `json:"backup_strategy,omitempty"`
```

Comment:
why do we delete this parameter?

Reply:
A new parameter is added to the BackupStrategy field, and a new API is used to create the BackupStrategy, so this parameter is unnecessary